packages/examples/cvat/recording-oracle/src/services/cloud/types.py

```diff
@@ -110,7 +111,9 @@ def from_url(cls, url: str) -> BucketAccessInfo:
                 path=parsed_url.path.lstrip("/"),
             )
         if Config.features.enable_custom_cloud_host:
-            if is_ipv4(parsed_url.netloc):
+            # Check if netloc is an ip address
+            # or localhost with port (or its /etc/hosts alias, e.g. minio:9000)
+            if is_ipv4(parsed_url.netloc) or re.fullmatch(r"\w+:\d{4}", parsed_url.netloc):
```

Comment:
@zhiltsov-max jfyi: it was done earlier for ExcO ([ref](https://github.com/humanprotocol/human-protocol/commit/9a48e1ee1e87324b659d45d1a2e61a0d74ad30ad)) but wasn't for RecO because didn't affect e2e flow locally; after some recent updates it was failing to load GT locally, so updated it for RecO as well here